### PR TITLE
Various: More wof:shortcodes, removed wof:abbreviations

### LINKED ISSUE #924: Create new wof:shortcode property
Once https://github.com/whosonfirst/whosonfirst-properties/pull/35 lands (and barring any changes to that logic), update records with `wof:abbreviation`.

Create a new `wof:shortcode` property using the `wof:abbreviation` property value.

Related: https://github.com/whosonfirst-data/whosonfirst-data/pull/844

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #924.

This PR:

- Moves `wof:abbreviation` property value to `wof:shortcode`
- Removes any `wof:abbreviation` property
- Includes changes from #823

List of updated ids: [list](https://github.com/whosonfirst-data/whosonfirst-data/files/2368632/389002b77f0cb.txt)

### CODE REVIEW

**Comment by nvkelso** [comment]
Please ensure the changes from #823 are incorporated here and then close that other PR in favor of this one.

**Comment by stepps00** [comment]
The most recent commits fix merge conflicts and include changes from #823. Ready for review.

File: data/101/748/361/101748361.geojson [hunk 1]
```diff
@@ -530,7 +530,6 @@
     "wd:longitude":-0.883333,
     "wd:population":664953,
     "wd:wordcount":3084,
-    "wof:abbreviation":"ZGZ",
```
Comment (nvkelso) [inline data/101/748/361/101748361.geojson]: Is this getting stashed somewhere?

File: data/101/748/459/101748459.geojson [hunk 2]
```diff
@@ -544,7 +544,6 @@
     "wd:longitude":7.591111,
     "wd:population":173619,
     "wd:wordcount":10187,
-    "wof:abbreviation":"EAP",
```
Comment (nvkelso) [inline data/101/748/459/101748459.geojson]: Is this getting stashed somewhere?

File: data/101/751/809/101751809.geojson [hunk 3]
```diff
@@ -378,7 +378,6 @@
     "wd:longitude":5.316667,
     "wd:population":143782,
     "wd:wordcount":2567,
-    "wof:abbreviation":"'s Bosch",
```
Comment (nvkelso) [inline data/101/751/809/101751809.geojson]: Is this getting stashed somewhere?

**Comment by nvkelso** [comment]
Please ensure the shortcode doesn't include punctuation (`US` not `U.S.`):

https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data/stepps00/more-shortcodes/data/856/337/93/85633793.geojson

**Comment by nvkelso** [comment]
And this region in Australia should be 3-char not 2-char:

https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data/stepps00/more-shortcodes/data/856/815/45/85681545.geojson

**Comment by nvkelso** [comment]
Other than that LGTM

File: data/101/748/361/101748361.geojson (hunk 1 above)
Comment (thisisaaronland) [reply data/101/748/361/101748361.geojson]: Abbreviations should probably follow the same convention as names and labels - the default is English and the specifics stored in `key:LANG` properties

File: data/101/751/809/101751809.geojson (hunk 3 above)
Comment (stepps00) [reply data/101/751/809/101751809.geojson]: Following up with abbreviations via: https://github.com/whosonfirst-data/whosonfirst-data/issues/1319

**Comment by stepps00** [comment]
181f8d1 - merged master into this branch
7f528fd - took care of comments about Australia and US records
14c1167 - brings shortcodes from any newly superseded record to the record superseding it

Ready for another review/merge, @nvkelso

**Comment by stepps00** [comment]
re https://github.com/whosonfirst-data/whosonfirst-data/pull/1055#commitcomment-30554696:

The correct thing to do here is removing the id from the coterminous property (or removing the coterminous property altogether).. records were cleaned up in 0bb4b89

Will QA a little more, then merge.

**Pull request merged**